### ISSUE TITLE
diff() has issues with slashes in object keys

### ISSUE THREAD

**Issue by hinzundcode** [idubrov/json-patch#9]
Great library, exactly what I needed! The only problem is I can't diff objects with keys containing slashes. The generated patch is invalid and can't be applied.

```rust
extern crate serde_json;
extern crate json_patch;
use serde_json::json;
use json_patch::{diff, patch};

fn main() {
	let mut json_a = json!({
		"/foo/bar": 1,
	});
	
	let json_b = json!({
		"/foo/bar": 2,
	});
	
	let p = diff(&json_a, &json_b);
	dbg!(&p);
	
	patch(&mut json_a, &p).unwrap();
	dbg!(&json_a);
}
```

```
[src/main.rs:16] &p = Patch(
    [
        Replace(
            ReplaceOperation {
                path: "//foo/bar",
                value: Number(
                    2,
                ),
            },
        ),
    ],
)
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidPointer', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by idubrov** [comment]
Should be fixed in `0.2.6`.

**Comment by hinzundcode** [comment]
thank you!
